tests/unit/lms/services/jstor_test.py

```diff
@@ -6,11 +6,14 @@
 
 from lms.models import ApplicationSettings
 from lms.services import ExternalRequestError
-from lms.services.jstor import JSTORService, factory
+from lms.services.jstor import ContentStatus, JSTORService, factory
 from tests import factories
 
 JSTOR_API_URL = "http://api.jstor.org"
 
+# Defaults for required fields in `/metadata/{doi}` responses.
+METADATA_RESPONSE_DEFAULTS = {"has_pdf": True, "requestor_access_level": "full_access"}
+
```

Comment:
Given the number of times this gets used, it might be worth working into a named fixture. A caller pattern like this would get you most of the way:

```
    @pytest.fixture
    def set_metadata_response(self, http_service):
        def set_metadata_response(values):
            json_data = {"has_pdf": True, "requestor_access_level": "full_access"}
            json_data.update(values)

            http_service.get.return_value = factories.requests.Response(
                json_data=json_data
            )

        return set_metadata_response
```

This doesn't completely solve it as one of the tests requires that fields be missing, but in that case doing it explicitly might be the way to go.